packages/ui-react/src/pages/Home/Home.tsx

```diff
@@ -3,18 +3,21 @@ import { shallow } from '@jwp/ott-common/src/utils/compare';
 import type { Content } from '@jwp/ott-common/types/config';
 import { useConfigStore } from '@jwp/ott-common/src/stores/ConfigStore';
 import { useTranslation } from 'react-i18next';
+import { useFilterContent } from '@jwp/ott-ui-react/src/hooks/useFilteredContent';
 
 import ShelfList from '../../containers/ShelfList/ShelfList';
 
 const Home = () => {
   const { config } = useConfigStore(({ config, accessModel }) => ({ config, accessModel }), shallow);
   const content: Content[] = config?.content;
   const { t } = useTranslation('common');
+  const labelsFilteringEnabled = Boolean(config?.custom?.enableLabelsFiltering);
```

Comment:
This one is always a string. Feel free to use a `isTruthyCustomParamValue` function.